app/javascript/src/components/Player.tsx

```diff
@@ -347,8 +332,10 @@ class Player extends React.Component<PlayerProps, PlayerState> {
   }
 
   private toggleDebugger() {
-    this.debug = !this.debug
-    this.debugRerender()
+    this.setState({
+      debug: !this.state.debug
+    },
+    undefined);
```

Comment:
You should be able to omit the second argument to `setState` (optional callback). I think there's one other spot in this file where that's also the case. [ref to the docs](https://reactjs.org/docs/react-component.html#setstate)